src/cascadia/TerminalApp/App.cpp

```diff
@@ -635,6 +635,10 @@ namespace winrt::TerminalApp::implementation
                 {
                     _RemoveTabViewItem(tabViewItem);
                 }
+                else if (p != nullptr && p->GetCloseOnExit())

```

Comment:
This can probably be folded into the above, and it will conflict with #737.

Reply:
If #737 merges first, it will be good to consolidate the `_lastTabClosedHandlers()` logic into the one function @bef178 added for that purpose.

Reply:
I'm going to merge #737 right now, so I'd make sure to move this logic into `_RemoveTabViewItem`